frontend/src/lib/components/common/form/Field.svelte

```diff
@@ -11,21 +13,27 @@ Meant to be mainly used with two chidlren that are aligned on both ends of the f
 ### Properties
 
 - `customStyle`: A custom style passed on to the parent component.
-
-You should not try to use a variant and customize at the same time.
+- `bgColor`: The background color of the field.
+- `id`: The id of the field used for labeling. Optional.
+- `label`: The label of the field. Label will not be shown if it's empty. Optional.
 
 ### Usage
 
 ```tsx
 <FieldGroup>
-  <Field>
-    <label for="input_1"> Label for input </label>  
+  <Field id="input_1" label="label">
     <input id="input_1"/>
   </Field>
 </FieldGroup>
 ```
 -->
 
 <div class="flex h-full items-center justify-between {bgColor}" style={customStyle}>
+  {#if id && label && label?.length > 0}
+    <label
+      for={id}
+      class="label-sm label pointer-events-none mx-6 my-2 whitespace-nowrap text-secondary"
```

Comment:
`whitespace-nowrap` is problematic here. The label should be able to wrap. Please use `grow` values for it and the contents or a fractional minimum width to get the proper behaviour.